docs/testcases.rst

```diff
@@ -394,18 +394,18 @@ Click in empty space to deselect the collapsed job.
 
 Click on the ( + ) global Expand/Collapse icon in the navbar to toggle all +n counts.
 
-    **Expected**: Counts should expand and collapse on all visible resultsets.
+    **Expected**: Counts should expand and collapse on all visible pushes.
 
 Navigate via the n,p and left/right keys.
 
     **Expected**: +n counts should be skipped during navigation.
 
 expand all the groups, (the url querystring will reflect this) then reload the page
 
-    **Expected**: groups should still be expanded for all resultsets
+    **Expected**: groups should still be expanded for all pushes
 
 Optional: There are other variants that can be tested: classification of expanded job count members, Filters, and any other workflow integration testing.
 
 .. _`stage`: https://treeherder.allizom.org
 .. _`production`: https://treeherder.mozilla.org
-.. _`userguide`: https://treeherder.allizom.org/userguide.html
\ No newline at end of file
+.. _`userguide`: https://treeherder.allizom.org/userguide.html
```

Comment:
Yeah - foo.allizom.org is the stage version of foo.mozilla.org. Currently the link points to stage, when it should point to production.